distribution/src/config/jvm.options

```diff
@@ -16,19 +33,23 @@
 ##
 ################################################################
 
-# Xms represents the initial size of total heap space
-# Xmx represents the maximum size of total heap space
+# Xms represents the initial size of the JVM heap
+# Xmx represents the maximum size of the JVM heap
 
 -Xms${heap.min}
 -Xmx${heap.max}
 
+
+
 ################################################################
 ## Expert settings
 ################################################################
 ##
-## All settings below this section are considered
-## expert settings. Don't tamper with them unless
-## you understand what you are doing
+## All settings below here are considered expert settings.  Do
+## not tamper with them unless you understand what you are
+## doing. If you want to adjust any of these settings then do
+## not do so by editing this file; instead, create a new file in
```

Comment:
The wording here is a bit confusing (when I first read it, I think it is *do so*, my mind accidentally skipped over the first "do not").  Could we simplify the first clause to simply `If you want to adjust any of these settings then do not edit this file;`?

Reply:
Good point, the line break ended up in a bad place there. I tried a few ideas and ended up copying the wording from the earlier section, see cd1b171.